tests/integration/test_yarn.py

```diff
@@ -0,0 +1,55 @@
+import logging
+from pathlib import Path
+
+import pytest
+
+from . import utils
+
+log = logging.getLogger(__name__)
+
+
+@pytest.mark.parametrize(
+    "test_params",
+    [
+        pytest.param(
+            utils.TestParameters(
+                repo="https://github.com/cachito-testing/cachi2-yarn-berry.git",
+                ref="76131443f7472916b316954c957259ee4cee97a7",
+                packages=({"path": ".", "type": "yarn"},),
+                check_vendor_checksums=False,
+                expected_exit_code=0,
+                expected_output="All dependencies fetched successfully",
+            ),
+            marks=pytest.mark.xfail,  # temporary
+            id="offline-cache",
```

Comment:
This isn't an [offline cache](https://v3.yarnpkg.com/features/offline-cache) test case

The offline cache is the .yarn/cache directory which contains all the dependencies. The second test case has the offline cache, this one doesn't. (The second one also has the .pnp.cjs file. Offline cache + .pnp.cjs = zero-installs.)